server.js

```diff
@@ -69,19 +70,38 @@ app.get("/historical/:country", async function (req, res) {
 });
 
 app.get("/countries/:country", async function (req, res) {
-  let countries = JSON.parse(await redis.get(keys.countries))
-  const standardizedCountryName = countryMap.standardizeCountryName(req.params.country.toLowerCase());
+  let countries = JSON.parse(await redis.get(keys.countries));
+
+  // @ebwinters version
+  // const standardizedCountryName = countryMap.standardizeCountryName(req.params.country.toLowerCase());
+  // let country = countries.find(e => {
+  //   // see if strict was even a parameter
+  //   if (req.query.strict) {
+  //     return req.query.strict.toLowerCase() == 'true' ? e.country.toLowerCase() === standardizedCountryName : e.country.toLowerCase().includes(standardizedCountryName)
+  //   }
+  //   else {
+  //     return e.country.toLowerCase().includes(standardizedCountryName);
+  //   }
+  // });
+
+  // @buster95 version
+  const countryData = country_utils.getCountryData(req.params.country);
   let country = countries.find(e => {
     // see if strict was even a parameter
     if (req.query.strict) {
-      return req.query.strict.toLowerCase() == 'true' ? e.country.toLowerCase() === standardizedCountryName : e.country.toLowerCase().includes(standardizedCountryName)
-    }
-    else {
-      return e.country.toLowerCase().includes(standardizedCountryName);
+      return req.query.strict.toLowerCase() == 'true' ?
+        e.country.toLowerCase() === countryData.country.toLowerCase() :
+        e.country.toLowerCase().includes(countryData.country)
+    } else {
+      if (countryData.country) {
+        return e.country.toLowerCase().includes(countryData.country.toLowerCase());
+      }
     }
   });
```

Comment:
Need to delete this

Reply:
@ebwinters I can't delete this remember, country names has changed in redis storage, names are differents than world meter country names, therefore your code dosen't work

Reply:
I tested locally and it worked fine

Reply:
Your code work but not for all countries